docs/api/session.md

```diff
@@ -980,12 +980,17 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
          and this is set to `true`, then local playback of audio will not be muted (e.g. using `MediaRecorder`
          to record `WebFrameMain` with this flag set to `true` will allow audio to pass through to the speakers
          while recording). Default is `false`.
+* `opts` Object (optional) _macOS_ _Experimental_
+  * `useSystemPicker` Boolean - true if a user wants to use the native system picker _macOS_ _Experimental_
```

Comment:
* Is this based on whether the _user_ wants to use the native system picker, or is it determined by the app developer?
* Is the default for this `false`?
* Can we specify the behaviour for macOS 14 and below?

Reply:
* It's determined by the app developer - the user won't be able to choose between the native picker vs a different picker, it will be set to whatever the developer decides (this also follows other generic/native capturer implementations, so it's fortunately not a different behavior 🙂 )
* Yes, it's `false` - I'll clarify that in the docs!
* Yep, good callout, I'll specify that the behavior will fall back to the existing `desktopCapturer.getSources` method when `useSystemPicker` is not supported 👍